utils/apiAuth.js

```diff
@@ -34,20 +35,15 @@ const addNewKey = async (req, res, next) => {
     };
     return res.status(201).json(returnKey);
   } catch (e) {
-    handleErrors(e, 'Error adding sign-off', logger, next);
+    handleErrors(e, 'Error adding API key', logger, next);
   }
 };
 
 const genAPIKey = async (user) => {
-  //create a base-36 string that contains 30 chars in a-z,0-9
-  const APIstring = [...Array(30)]
-    .map((e) => ((Math.random() * 36) | 0).toString(36))
-    .join('');
-
-  const APIident = [...Array(30)]
-    .map((e) => ((Math.random() * 36) | 0).toString(36))
-    .join('');
+  const APIident = uid(36);
```

Comment:
From a quick look at the docs, it seems this caches UIDs in memory and will lose its cache on a restart, so it can still generate duplicates. It's very unlikely though, and since there's a unique constraint at the DB, it would just throw an exception, and the request could be resent, so I think that's fine